backend/src/main/kotlin/org/loculus/backend/controller/SubmissionController.kt

```diff
@@ -108,12 +116,15 @@ open class SubmissionController(
                 innerDataUseTermsType = dataUseTermsType
             }
         }
-
+        val fileMappingParsed = fileMapping?.let {
```

Comment:
Should probably throw BadRequestException here saying that the fileMapping could not be parsed.